components/BottomSheet/src/MDCBottomSheetController.m

```diff
@@ -146,6 +149,16 @@ - (void)setDismissOnBackgroundTap:(BOOL)dismissOnBackgroundTap {
   self.mdc_bottomSheetPresentationController.dismissOnBackgroundTap = dismissOnBackgroundTap;
 }
 
+- (BOOL)dismissOnDraggingDownSheet {
```

Comment:
Shouldn't we be using self here?

Reply:
i followed the consistency of the rest of the APIs, and have this be a computed variable instead of a stored variable. what value do you see by adding another storage in this class?

Reply:
My concern here is we should move the rest of the file up to the "correct" way instead of introducing more that isn't the "correct" way.

Reply:
I am still not sure that what it has right now is incorrect, which is why I am questioning this change. Let's follow up on this if we want to change the way the class passes properties down the chain.